elements/urth-core-behaviors/jupyter-widget-behavior.html

```diff
@@ -6,6 +6,13 @@
 <!--
 This behavior is used to encapsulate some of the services needed for ipywidgets.
 -->
+<style>
+    .output_area pre.urth-widget-error {
+        color: red;
+        background-color: #fdd;
+        padding: 5px;
+    }
+</style>
```

Comment:
@deedubbu What do you think about these styles here? The good news is that they are only used in this file, so maybe it's the most simple and elegant thing to do.